src/main/java/edu/kis/powp/jobs2d/drivers/JaneDriverManager.java

```diff
@@ -0,0 +1,25 @@
+package edu.kis.powp.jobs2d.drivers;
+
+import edu.kis.powp.jobs2d.AbstractDriver;
+import edu.kis.powp.jobs2d.Job2dDriver;
+import edu.kis.powp.jobs2d.LoggerDriver;
+
+public class JaneDriverManager extends AbstractDriver {
+    private Job2dDriver currentDriver = new LoggerDriver();
```

Comment:
It is better to use constructor injection to avoid dependency with particular implementation